### ISSUE TITLE
Add currency conversion to each transaction

### ISSUE THREAD

**Issue by kenzieschmoll** [terra-money/finder#64]
Conversion should be shown in a user-configurable selected currency. The transaction amounts should be converted at the conversion rate for the timestamp of the transaction. 

Example for an airdrop transaction:
<img width="254" alt="Screen Shot 2021-03-17 at 8 13 22 AM" src="https://user-images.githubusercontent.com/43759233/111490971-af89ff00-86f8-11eb-8ca0-859d095aefec.png">
add another line below that shows "total amount in USD" (where USD in this example is the user-selected currency. You can imagine a setting or a selector that says "show transaction amount in USD", "show transaction amount in UST", etc.). The total amount in USD would be the value of the tokens converted to USD at the conversion rate for the transaction timestamp.

Example for withdrawal transaction:
<img width="696" alt="Screen Shot 2021-03-17 at 8 09 37 AM" src="https://user-images.githubusercontent.com/43759233/111490713-7b164300-86f8-11eb-969e-f7149ef2c3a0.png">
add another line that show "total amount in USD"

This issue is related to https://github.com/terra-project/station/issues/141 in that users will need tools to track rewards, airdrops, sales, etc. This extra "conversion" data could be surfaced in the Finder UI, or even better, surfaced at the API level so that not only Finder can use it, but also other tools that want to leverage the same APIs.

**Comment by hanjukim** [comment]
Thank you for your proposal. We will add this feature.

**Comment by kenzieschmoll** [comment]
Excellent - surfacing this in the API would be ideal. I am currently working on some tax software using the terra apis and am having to pull in another api (CoinDesk) to manually get the conversion rate for currencies. CoinDesk has request limits, so this is causing some issues. If the total amount in a specified currency (perhaps set by a query parameter on the txs request) was provided by the terra api, using another API to get that data would not be necessary and would be helpful for other tools that want to consume this data.

**Comment by ALPAC-4** [comment]
We discussed this proposal and we decided not do this. Here are reason of this.

What we need to do this?
We need historical price data for all natvie/cw20/IBC token. To show actual value of that moment(when the tx executed) we need to collect all of the token's price for all time range.

Hard point of this.
- There are tons of tokens on chain: This make hard to collect price of new token.
- There is token that doesn't have a price: Some tokens are not have DEX pair or UST pair.

For these reasons we decided not doing this.

And sorry for the late comment.
